code/controllers/subsystem/traumas.dm

```diff
@@ -20,35 +20,80 @@ SUBSYSTEM_DEF(traumas)
 						"greytide"  = strings(PHOBIA_FILE, "greytide"),
 						"lizards"   = strings(PHOBIA_FILE, "lizards"),
 						"skeletons" = strings(PHOBIA_FILE, "skeletons"),
-						"snakes"	= strings(PHOBIA_FILE, "snakes")
+						"snakes"	= strings(PHOBIA_FILE, "snakes"),
+						"robots"	= strings(PHOBIA_FILE, "robots"),
+						"doctors"	= strings(PHOBIA_FILE, "doctors"),
+						"authority"	= strings(PHOBIA_FILE, "authority"),
+						"the supernatural"	= strings(PHOBIA_FILE, "the supernatural")
 					   )
 
 	phobia_mobs = list("spiders"  = typecacheof(list(/mob/living/simple_animal/hostile/poison/giant_spider)),
 					   "security" = typecacheof(list(/mob/living/simple_animal/bot/secbot)),
 					   "lizards"  = typecacheof(list(/mob/living/simple_animal/hostile/lizard)),
-					   "snakes"   = typecacheof(list(/mob/living/simple_animal/hostile/retaliate/poison/snake))
+					   "snakes"   = typecacheof(list(/mob/living/simple_animal/hostile/retaliate/poison/snake)),
+					   "robots"   = typecacheof(list(/mob/living/silicon/robot, /mob/living/silicon/ai, 
+					   /mob/living/simple_animal/drone, /mob/living/simple_animal/bot/)),
+					   "the supernatural"   = typecacheof(list(/mob/living/simple_animal/hostile/construct, 
+					   /mob/living/simple_animal/hostile/clockwork/, /mob/living/simple_animal/drone/cogscarab))
 					   )
```

Comment:
can you add skeleton simplemobs to this list i keep forgetting to fix this sorry

Reply:
Sure no problem